.tekton/client-server-cg-pull-request.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "main" && ( "Dockerfile.client-server-cg.rh".pathChanged() || ".tekton/client-server-cg-pull-request.yaml".pathChanged() || "trigger-konflux-builds.txt".pathChanged() )
+    pipelinesascode.tekton.dev/task: "[.tekton/cosign-unit-test.yaml]"
```

Comment:
This should only be needed in `.tekton/cosign-pull-request.yaml` and `.tekton/cosign-pull-request.yaml`

Reply:
Oh ok thank you, I removed it in the other ones :)